scripts/stablex/place_spread_orders.js

```diff
@@ -0,0 +1,128 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const fetch = require("node-fetch")
+const BN = require("bn.js")
+const readline = require("readline")
+
+const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+const token_list_url = "https://raw.githubusercontent.com/gnosis/dex-js/master/src/tokenList.json"
+
+const promptUser = function(message) {
+  return new Promise(resolve => rl.question(message, answer => resolve(answer)))
+}
+
+const fetchTokenInfo = async function() {
+  console.log(`Recovering token data from URL ${token_list_url}`)
+  const token_list = await (await fetch(token_list_url)).json()
+  const token_data = {}
+  for (const token of token_list) {
+    token_data[token.id] = token
+  }
+  return token_data
+}
+
+const formatAmount = function(amount, token) {
+  return new BN(10).pow(new BN(token.decimals)).muln(amount)
+}
+
+const argv = require("yargs")
+  .option("tokens", {
+    alias: "t",
+    type: "array",
+    describe: "Collection of trusted tokenIds",
+    coerce: array => {
+      return array.flatMap(v => v.split(",").map(t => parseInt(t)))
+    },
+  })
+  .option("accountId", {
+    describe: "Account index of the order placer",
+  })
+  .option("spread", {
+    type: "float",
+    describe: "Percentage increase required for trade (fees not accounted)",
+    default: 0.25,
+  })
+  .option("sellAmount", {
+    type: "int",
+    describe: "Maximum sell amount (in full token units)",
+    default: 1000,
+  })
+  .option("validFrom", {
+    type: "int",
+    describe: "Number of batches (from current) until order become valid",
+    default: 3,
+  })
+  .option("expiry", {
+    type: "int",
+    describe: "Maximum auction batch for which these orders are valid",
+    default: 2 ** 32 - 1,
+  })
+  .demand(["tokens", "accountId"])
+  .help(
+    "Make sure that you have an RPC connection to the network in consideration. Example usage \n   npx truffle exec scripts/stablex/place_spread_orders.js --tokens=2,3,4 --accountId 0 --spread 0.3 --validFrom 5"
```

Comment:
Maybe tell the user how the network configuration is specified here?

Reply:
Not entirely sure what to do about this... I would almost rather just set help to false.